PrebidFledge.md

```diff
@@ -0,0 +1,185 @@
+# Prebid Fledge
+
+An exploration of the role Prebid can play in a Fledge world.
+
+## Fledge and Ad Selection
+
+While Fledge proposes a number of changes to how ad tech will function, this document is focused on the changes to the ad selection process in the context of the browser.
+
+### Chrome/Fledge Becomes the Final Selection Layer
+
+Today GAM is the final ad selection layer because it contains the publisher's direct demand, is tightly integrated with AdX, and doesn't provide a price signal externally. However, in a Fledge world, Chrome will contain interest group demand that isn't accessible via any other means and won't provide a price signal externally. When ```navigator.runAdAuction``` returns 'true' it has identified an appropriate ad for rendering based on provided bids and scoring logic. As a result, the browser/Fledge must become the final selection layer.
```

Comment:
```suggestion
Today GAM is the final ad selection layer because it contains the publisher's direct demand, is tightly integrated with AdX and doesn't provide a price signal externally. However, in a Fledge world, Chrome will contain interest group demand that isn't accessible via any other means and won't provide a price signal externally. When `navigator.runAdAuction` returns 'true' it has identified an appropriate ad for rendering based on provided bids and scoring logic. As a result, the browser/Fledge must become the final selection layer.
```